src/main/java/com/adyen/model/marketpay/notification/GenericResponse.java

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
We already have a GenericResponse.java in: 
src/main/java/com/adyen/model/GenericResponse.java

Is this one different/necessary?